src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/FeatureFlagsDebugPass.php

```diff
@@ -9,30 +9,22 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Bundle\FeatureFlagsBundle\DependencyInjection\CompilerPass;
+namespace Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler;
 
-use Symfony\Bundle\FeatureFlagsBundle\Debug\TraceableFeatureChecker;
-use Symfony\Bundle\FeatureFlagsBundle\Debug\TraceableStrategy;
 use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Reference;
+use Symfony\Component\FeatureFlags\Debug\TraceableFeatureChecker;
+use Symfony\Component\FeatureFlags\Debug\TraceableStrategy;
 
-final class DebugPass implements CompilerPassInterface
+final class FeatureFlagsDebugPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container): void
     {
         if (!$container->has('feature_flags.data_collector')) {
             return;
         }
 
-        $container->register('debug.feature_flags.feature_checker', TraceableFeatureChecker::class)
```

Comment:
moved ?

Reply:
Moved in src/Symfony/Bundle/FrameworkBundle/Resources/config/feature_flags_debug.php